ha/k8s_setup/ha_setup.py

```diff
@@ -216,7 +222,9 @@ def process(self):
                          'FAULT_TOLERANCE' : {'message_type' : 'cluster_event', 'consumer_group' : 'event_listener',
                                               'consumer_id' : '1'},
                          'NODE': {'resource_type': 'node'},
-                         'SYSTEM_HEALTH' : {'num_entity_health_events' : 2}
+                         'SYSTEM_HEALTH' : {'num_entity_health_events' : 2},
+                         'sys_health_bootstrap_timeout' : timeout,
```

Comment:
can it be the child of 'SYSTEM_HEALTH' or it should be outside of it?

Reply:
Moved into SYSTEM_HEALTH

Reply:
please mark resolved